### DISCUSSION TITLE
Datasource item children in disconnected mock data

### DISCUSSION THREAD

**Discussion by vitaliitylyk** [Sitecore/jss#1398] [Ideas]
## Description

The Manifest API documentation (https://jss.sitecore.com/docs/techniques/working-disconnected/manifest-api) describes that is is possible to add child items to datasources (`children: array[route]`), however this does not seem to work. 

Same for `children` on the route level. Looks like either documentation is wrong, or functionality is broken. 

## Expected behavior

Please update the documentation on how to mock children of the datasource items, although @aweber1 has mentioned in the Slack conversation that this functionality is not implemented.  

And if this functionality does not exist, please consider this issue as a *Feature Request* as well, as it is very important to be able to mock item hierarchies in disconnected mode.

**Comment by adoprog** [comment]
Has it ever been fixed / implemented? Is there any other way to mock datasource children in disconnected mode? This seems to be quite useful for CMP integration.
